src/js/components/Select/README.md

```diff
@@ -311,7 +311,7 @@ Required. Options can be either a string or an object. If an object is used, use
 
 **open**
 
-Initial state of the select component
+Control the state of the component.
```

Comment:
what was wrong with the previous description? I feel like `Control the state of the component.` doesn't cover the description better, unless I'm missing something.

Reply:
From the discussion [here](https://github.com/grommet/grommet/issues/2926#issuecomment-473341079) i thought that it is best to change the description. Not sure how to describe though. Open for suggestions.

Reply:
me too. let me think about it a little.

Reply:
Something like "Whether the state of the component should be open" ?